server/src/main/java/io/spine/server/entity/RecordBasedRepository.java

```diff
@@ -249,7 +251,8 @@ protected E findOrCreate(I id) {
         final EntityQuery<I> entityQuery = EntityQueries.from(filters, getEntityClass());
         final EntityQuery<I> completeQuery = toCompleteQuery(entityQuery);
         final Iterator<EntityRecord> records = recordStorage().readAll(completeQuery, fieldMask);
-        final Iterator<E> result = transform(records, storageRecordToEntity());
+        final Function<EntityRecord, E> transformer = entityConverter().reverse();
```

Comment:
Pease rename the variable `transformer` into the kind of transformation. Do not suggest `OptimusPrime` or something :) Tell what the transformation does.